demisto_sdk/commands/test_content/execute_test_content.py

```diff
@@ -40,7 +44,10 @@ def _add_pr_comment(comment, logging_module):
                 response = requests.get(issue_url, headers=headers, verify=False)
                 issue_comments = _handle_github_response(response, logging_module)
                 for existing_comment in issue_comments:
-                    if SKIPPED_CONTENT_COMMENT in existing_comment.get('body'):
+                    if is_skipped_tests_flow and SKIPPED_CONTENT_COMMENT in existing_comment.get('body'):
```

Comment:
```suggestion
                    if is_skipped_tests_flow and SKIPPED_CONTENT_COMMENT in existing_comment.get('body',''):
```

if `body` is always a key of `existing_comment`, you can use `comment['body']`, otherwise, put a default so it doesn't check `if SKIPPED_CONTENT_COMMENT in None` and crash